### PR TITLE
grid was implemented

### PR DESCRIPTION
**Pull request by superpooperxxx**
- [DEMO LINK](https://superpooperxxx.github.io/layout_catalog_grid/)

### CODE REVIEW

**Review by Fynkzia — APPROVED** [review]
I think everythink looks good

File: src/index.html [hunk 1]
```diff
@@ -17,62 +17,294 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
-        <img src="./images/logo.png" alt="Moyo logo">
+      <a href="#home" class="logo">
+        <img 
+          src="./images/logo.png" 
+          alt="MOYO logo"
```
Comment (solaryasha) [inline src/index.html:23]:
```suggestion
          alt="logo of the MOYO website" 
```

File: src/styles/blocks/_nav.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    height: 60px;
+
+    font-size: 12px;
+    font-weight: 500;
+    text-transform: uppercase;
+    color: $color-main-accent;
+
+    transition: color $transition-duration;
+
+    &--active {
```
Comment (solaryasha) [inline src/styles/blocks/_nav.scss:19]: 🔥

File: src/styles/blocks/_stars.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+  &__star {
+    @include square(16px);
+    margin-right: 4px;
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  @for $index from 1 through 5 {
```
Comment (solaryasha) [inline src/styles/blocks/_stars.scss:12]: 🔥

**Review by solaryasha — APPROVED** [review]
Nice job, Kyrylo !

**Pull request closed without merging**